packages/account/src/Components/inline-note-with-icon/inline-note-with-icon.tsx

```diff
@@ -1,8 +1,16 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { HTMLAttributes } from 'react';
 import { Icon, Text } from '@deriv/components';
 
-const InlineNoteWithIcon = ({ icon, message, title }) => {
+type TInlineNoteWithIconExtend = {
+    icon: string;
+    message: string;
+    title: string;
+};
+
+export type TInlineNoteWithIcon = HTMLAttributes<HTMLInputElement | HTMLLabelElement> &
```

Comment:
just for my information, why did you add `HTMLAttributes<HTMLInputElement | HTMLLabelElement>` ?because I checked and without having it there is no error